### PR TITLE
Initial Project Folder

### PR DESCRIPTION
**Pull request by rahulcs754**
1. Initial Project folder
2. added folder for pages
3. added routes for pages
image
image
image
image
image
image

### CODE REVIEW

**Review by coldpigli — APPROVED** [review]
Only Routing Component added.
Everything looks good. Liked how you created a new component for routes.

File: src/App.css [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+.brand-description {
+  font-size: 1.4rem;
+}
+
+.links {
+  display: flex;
+}
+
+.links a {
+  color: #d9d9d9;
```
Comment (rdshinde) [inline src/App.css:48]: You could have used variables for colours.

File: src/Pages/index.js [hunk 2]
```diff
@@ -0,0 +1,9 @@
+import {Home} from "./Home";
+import {History} from "./History";
+import {Login} from "./Login";
+import {Signup} from "./Signup";
+import {VideoList} from "./VideoList";
+import {WatchLater} from "./WatchLater";
+import {NoPageFound}from "./Error";
+
+export {Home,History,Login,Signup,VideoList,WatchLater,NoPageFound};
```
Comment (rdshinde) [inline src/Pages/index.js:9]: We can use export directly instead of first importing and then exporting. 
For example `export {Home} from "./Home"` .

**Review by rdshinde — APPROVED** [review]
Routing seems fine.

File: src/App.css (hunk 1 above)
Comment (rahulcs754) [reply src/App.css:48]: okay sure

**Pull request merged**